### PR TITLE
Add Troubleshooting Steps for chocolateyguicli Command

### LINKED ISSUE #357: Add troubleshooting for when chococlateyguicli command isn't registered error happens
# Description
Need to go in and add troubleshooting steps to take when getting the error that the `chocolateyguicli` command is not registered. 

Needs added to either [this page](https://docs.chocolatey.org/en-us/chocolatey-gui/known-issues) in the docs. Or on a Chocolatey GUI FAQ section. 

## References:
* [ZenDesk Ticket](https://chocolatey.zendesk.com/agent/tickets/12922)

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
- Fixed relevant markdown linting errors on page.
- Added troubleshooting steps to take when seeing an error with the chocolateyguicli command. 

## Motivation and Context
Update and enhance documentation.

## Testing
Built site locally to preview changes made. 

## Change Types Made
* [ ] Bug fix (non-breaking change)
* [ ] Feature / Enhancement (non-breaking change)
* [ ] Breaking change (fix or feature that could cause existing functionality to change)
* [ ] PowerShell code changes.
* [X] Documentation update or addition

## Related Issue

Fixes #357 

## Change Checklist

* [X] Requires a change to the documentation
* [X] Documentation has been updated
* [ ] Tests to cover my changes, have been added
* [ ] All new and existing tests passed.
* [ ] PowerShell v2 compatibility checked.

### CODE REVIEW

File: input/en-us/chocolatey-gui/known-issues.md [hunk 1]
```diff
@@ -41,4 +39,14 @@ However, if you attempt the same operation via Chocolatey GUI, using the "Pin (I
...
\ No newline at end of file
+This will be addressed in a future version of Chocolatey CLI/Chocolatey GUI.
+
+## chocolateyguicli Command Not Registered by System
+
+When recieveing an error in shell saying the chocolateyguicli is not registered or `ObjectNotFound`. Please run through the following troubleshooting steps.
+
+Step 1: Reinstall the chococlateygui package. `choco upgrade chocolateygui -y --force`
+
+Step 2: If a re-install of the package did not fix the issue. Please check if you are running an AV solution, that it is not blocking the write action being taken on the Chocolatey GUI Config. Sometimes AV solutions will block this request from happening.
```
Comment (pauby) [inline input/en-us/chocolatey-gui/known-issues.md:50]:
```suggestion
1. If step 1 did did not fix the issue, please check that your antivirus software is not blocking the Chocolatey GUI Config file from being written to.
```

Comment (pauby) [reply input/en-us/chocolatey-gui/known-issues.md:50]: Can you add the location where the Chocolatey GUI config is? So:

```
1. If step 1 did did not fix the issue, please check that your antivirus software is not blocking the Chocolatey GUI Config file at <PATH TO CONFIG FILE> from being written to.
```

File: input/en-us/chocolatey-gui/known-issues.md [hunk 2]
```diff
@@ -14,22 +14,20 @@ This issue was tracked in this [GitHub issue](https://github.com/chocolatey/Choc
...
 * Remove the users from `Network Configuration Operators` - PowerShell offers an alternative to `ipconfig /flushdns` that does not require admin permissions - `Clear-DnsClientCache`.
 * OR change the ChocolateyGui.exe.manifest file in v0.17.0+ in the Chocolatey GUI folder under Program Files to `<requestedExecutionLevel level="asInvoker" uiAccess="false" />` (from `highestAvailable`).
 
-Please see for more details: https://github.com/chocolatey/ChocolateyGUI/issues/629
-
+Please see the following [GitHub Issue](https://github.com/chocolatey/ChocolateyGUI/issues/629) for more details.
 
 ## Pin command does not work when running with Background Mode enabled
 
 In the current versions of Chocolatey CLI (v0.11.1) and Chocolatey GUI (v0.19.0), when attempting to pin a package using the Chocolatey CLI (choco.exe) with Background Mode enabled, things will work as expected.  i.e. when running:
```
Comment (pauby) [inline input/en-us/chocolatey-gui/known-issues.md:29]:
```suggestion
In the current versions of Chocolatey CLI (v0.11.1) and Chocolatey GUI (v0.19.0), when attempting to pin a package using the Chocolatey CLI (choco.exe) with Background Mode enabled, things will work as expected.  i.e. when running `choco pin add --name="procmon"`.

```

File: input/en-us/chocolatey-gui/known-issues.md [hunk 3]
```diff
@@ -14,22 +14,20 @@ This issue was tracked in this [GitHub issue](https://github.com/chocolatey/Choc
...
 
-```
+```powershell
 choco pin add --name="procmon"
 ```
```
Comment (pauby) [inline input/en-us/chocolatey-gui/known-issues.md:32]:
```suggestion
```

**Review by pauby — DISMISSED** [review]

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**